hardhat.config.ts

```diff
@@ -115,15 +111,20 @@ const config: HardhatUserConfig = {
     avalanche: createConfig("avalanche"),
     arbitrum: createConfig("arbitrum"),
     optimism: createConfig("optimism"),
-  },
+  }, 
   paths: {
     artifacts: "./artifacts",
     cache: "./cache",
     sources: "./contracts",
     tests: "./test",
   },
   etherscan: { 
-     apiKey: getScanApiKey(String(process.env.networkType)),
+     apiKey: {
+       mainnet: String(ETHERSCAN_API),
+       polygon: String(POLYGONSCAN_API),
+       arbitrumOne: String(ARBISCAN_API),
+       avalanche: String(SNOWTRACE_API),
```

Comment:
@pulkitkejriwal1411 also add optimism and fantom api key

Reply:
Resolved.